ThunderCloud/DateFormatter+Extensions.swift

```diff
@@ -11,13 +11,35 @@ import Foundation
 extension DateFormatter {
     
     /// Standard `iso8601` `DateFormatter`
-    /// Note milliseconds including, for customization over `dateFormat`, see `iso8601(dateFormat:)`
-    static let iso8601: DateFormatter = {
+    /// Note milliseconds included, for customization over `dateFormat`, see `iso8601(dateFormat:timeZone:)`
+    static var iso8601: DateFormatter {
```

Comment:
This can be carefully migrated the `ThunderBasics` now taking account for `TimeZone` override if needed